doc/modules/manifold.rst

```diff
@@ -293,24 +293,24 @@ It requires ``n_neighbors > n_components * (n_components + 3) / 2``.
 
 .. dropdown:: Complexity
 
-The HLLE algorithm comprises three stages:
+  The HLLE algorithm comprises three stages:
 
-1. **Nearest Neighbors Search**.  Same as standard LLE
+  1. **Nearest Neighbors Search**.  Same as standard LLE
 
-2. **Weight Matrix Construction**. Approximately
-   :math:`O[D N k^3] + O[N d^6]`.  The first term reflects a similar
-   cost to that of standard LLE.  The second term comes from a QR
-   decomposition of the local hessian estimator.
+  2. **Weight Matrix Construction**. Approximately
+    :math:`O[D N k^3] + O[N d^6]`.  The first term reflects a similar
+    cost to that of standard LLE.  The second term comes from a QR
+    decomposition of the local hessian estimator.
```

Comment:
doc CI is still red as far as I can see, maybe try this (one more space so that `**Weight Matrix` aligns vertically with the following lines that belong to the same bullet point)?
```suggestion
     :math:`O[D N k^3] + O[N d^6]`.  The first term reflects a similar
     cost to that of standard LLE.  The second term comes from a QR
     decomposition of the local hessian estimator.
```

Reply:
The CI was showing me a big circle for a very long time. And most of the icons were blurred for some reason. Made fixes.